app/main.py

```diff
@@ -1,3 +1,5 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    lower_pharase = phrase.lower()
```

Comment:
There is a typo in the variable name `lower_pharase`. It should be `lower_phrase` to match the intended meaning.